tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -148,6 +148,11 @@ public void CloseSpan(Span span)
                 }
             }
 
+            if (string.Compare(span.ServiceName, Tracer.DefaultServiceName, StringComparison.Ordinal) != 0)
```

Comment:
Should this be `StringComparison.OrdinalIgnoreCase`? And `string.Equals` is probably preferable
```suggestion
            if (string.Equals(span.ServiceName, Tracer.DefaultServiceName, StringComparison.OrdinalIgnoreCase))
```

Reply:
Fixed see 14f828b6ca89a2f776967e35a647b6493ec40c97